### ISSUE TITLE
Carthage build: Task failed with exit code 70, using XCode 10.1 Swift 4.2 Carthage 0.31.2

### ISSUE THREAD

**Issue by aalenliang** [dennisweissmann/DeviceKit#158]
Cartfile
```
github "dennisweissmann/DeviceKit" ~> 1.3
```

Cartfile.resolved
```
github "dennisweissmann/DeviceKit" "1.10.0"
```

running command
```
carthage build
```

output:
```
*** Building scheme "DeviceKit" in DeviceKit.xcworkspace
Build Failed
	Task failed with exit code 70:
	/usr/bin/xcrun xcodebuild -workspace /.../Carthage/Checkouts/DeviceKit/DeviceKit.xcworkspace -scheme DeviceKit -configuration Release -derivedDataPath /.../Library/Caches/org.carthage.CarthageKit/DerivedData/10.1_10B61/DeviceKit/1.10.0 -sdk watchsimulator -destination platform=watchOS\ Simulator,id=88698357-C299-4CBC-9D30-01A3C3B2D36A -destination-timeout 3 ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES build (launched in /.../Carthage/Checkouts/DeviceKit)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/l0/cl5pmxqs02jgkt6b2g1gbggh0000gn/T/carthage-xcodebuild.PxD46Q.log
```

carthage-xcodebuild.PxD46Q.log
```
Build settings from command line:
    CARTHAGE = YES
    CODE_SIGN_IDENTITY =
    CODE_SIGNING_REQUIRED = NO
    ONLY_ACTIVE_ARCH = NO
    SDKROOT = watchsimulator5.1

--- xcodebuild: WARNING: Unable to open project file '/.../Carthage/Checkouts/DeviceKit/Example/DeviceKitPlayground.playground' in workspace '/.../Carthage/Checkouts/DeviceKit/DeviceKit.xcworkspace'.
xcodebuild: error: The run destination Apple Watch Series 2 - 38mm is not valid for Running the scheme 'DeviceKit'.
	Apple Watch Series 2 - 38mm doesn’t match any of DeviceKit.framework’s targeted device families. You can expand DeviceKit.framework’s targeted device families to support Apple Watch Series 2 - 38mm.
```

**Comment by aalenliang** [comment]
Realized i only need build for iOS, so I change the command to
```
carthage build --platform iOS
```
and now it works.

I checked the Target Device Family section of DeviceKit, it is not included watchOS. Don't know why carthage build this. I thinks this maybe a issue with Carthage.

**Comment by ifndefgt** [comment]
Can you please reopen the issue, I'm facing the same problem and I can't remove the carthage build target watchOS. This issue might be related to the warning. I suspect that, when carthage can't open the project it may fallback to defaults.

```WARNING: Unable to open project file '/.../Carthage/Checkouts/DeviceKit/Example/DeviceKitPlayground.playground' in workspace '/.../Carthage/Checkouts/DeviceKit/DeviceKit.xcworkspace'.```

When I open the xcworkspace in Xcode `DeviceKit.xcodeproj` is not accesible, it seems disabled.

I can successfully build version 1.8.0 on the same environment.

**Comment by dennisweissmann** [comment]
Sorry for that :( are you using the latest version? This should be fixed with #160 and version 1.11.0

**Comment by ifndefgt** [comment]
Yes I'm using 1.11.0 and the output is still same. 

```
Build settings from command line:
    CARTHAGE = YES
    CODE_SIGN_IDENTITY = 
    CODE_SIGNING_REQUIRED = NO
    ONLY_ACTIVE_ARCH = NO
    SDKROOT = watchsimulator5.1

--- xcodebuild: WARNING: Unable to open project file '.../Project/Carthage/Checkouts/DeviceKit/Example/DeviceKitPlayground.playground' in workspace '.../Project/Carthage/Checkouts/DeviceKit/DeviceKit.xcworkspace'.
xcodebuild: error: The run destination Apple Watch Series 2 - 38mm is not valid for Running the scheme 'DeviceKit'.
	Apple Watch Series 2 - 38mm doesn’t match any of DeviceKit.framework’s targeted device families. You can expand DeviceKit.framework’s targeted device families to support Apple Watch Series 2 - 38mm.
```
